include/seqan3/core/parallel/detail/spin_delay.hpp

```diff
@@ -72,6 +75,10 @@ class spin_delay
     {
         #if defined(__SSE2__)  // AMD and Intel
             _mm_pause();
+	#elif defined(__armel__) || defined(__ARMEL__) // armel, but broken
```

Comment:
it seems the alignment to the other statements is off by 2 spaces

Reply:
Arggh.. spaces vs. tabs :-P